addons/website_sale_stock/models/sale_order_line.py

```diff
@@ -6,11 +6,11 @@
 class SaleOrderLine(models.Model):
     _inherit = 'sale.order.line'
 
-    def _set_shop_warning_stock(self, desired_qty, new_qty):
+    def _set_shop_warning_stock(self, desired_qty, new_qty, product_name='product'):
```

Comment:
it's not usually accepted to modify the signature of method (https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean)

if it was really need, we usually use a context key to pass the information, or more rarely we could create a new method (and remove the old one that's not used anymore in master)

in this case it seems that it's not necessary since you could just use `self.product_id.name`

Reply:
makes sense, i will use the context instead.